### DISCUSSION TITLE
How to build harbour core with embarcadero 7.60 for windows 64 bits

### DISCUSSION THREAD

**Discussion by antonioalmaraz** [harbour/core#274] [General]
Haste el momento me fiunciona bien con 32 bits pero he intentado crear archivos bat para poder compilarlo sin exito a 64 bits 
alguien tiene algun script o instrucciones que me puedan compartir para poder compilarlo con esa version, en fivetech y su fork de harbour vi algo pero no funciona, agradezco sus notas se que gcc es gratuito y ya esta funcional pero estoy muy acostumbrado a lo que ante era borland

**Comment by FiveTechSoft** [comment]
Estimado Antonio,

Enviame un email a ***@***.*** y te ayudo

saludos

El mié, 16 nov 2022 a las 20:13, antonioalmaraz ***@***.***>)
escribió:

> Haste el momento me fiunciona bien con 32 bits pero he intentado crear
> archivos bat para poder compilarlo sin exito a 64 bits
> alguien tiene algun script o instrucciones que me puedan compartir para
> poder compilarlo con esa version, en fivetech y su fork de harbour vi algo
> pero no funciona, agradezco sus notas se que gcc es gratuito y ya esta
> funcional pero estoy muy acostumbrado a lo que ante era borland
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/harbour/core/discussions/274>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOC2UOQY2UHFX2ODQSEFM3WIUW7DANCNFSM6AAAAAASCSKKTA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Antonio Linares
www.fivetechsoft.com
